corehq/apps/app_manager/xform.py

```diff
@@ -1131,7 +1132,7 @@ def _get_select_question_option(item):
                         "tag": "hidden",
                         "value": '{}/@{}'.format(path, attrib),
                         "repeat": node_info['repeat'],
-                        "group": node_info['repeat'],
+                        "group": node_info['group'],
```

Comment:
Was this causing a problem previously or just being ignored?

Reply:
I don't think anything was using this information previously...